dev/test/main.py

```diff
@@ -0,0 +1,3 @@
+from art.skypilot import SkyPilotBackend
+
+backend = SkyPilotBackend()
```

Comment:
What is this testing? Just imports, or actual cluster initialization? I think we should adjust the file name to reflect what it tests.